app/views/gallery.scala.html

```diff
@@ -15,20 +14,40 @@
     <div id="gallery">
         <div class="sidebar">
             <div id="card-filter">
+                <div id="ribbon-menu-holder">
```

Comment:
There are actually two HTML elements with the same ID now. If you want them to have the same CSS, then you can create a new class for them, and just make sure that they have different IDs. Though the name `ribbon-menu-holder` doesn't really fit anyway, because I don't think that this counts as a ribbon menu? So maybe switching up the class name is a good idea too ;)